src/components/events/EventParkingMap.tsx

```
@@ -0,0 +1,219 @@
+        <div className="flex items-center gap-1.5 bg-white border border-black px-2.5 py-1 rounded text-xs font-bold text-gray-800">
+          <ShieldCheck className="w-4 h-4 text-emerald-600" />
+          <span>Official Event Parking</span>
+        </div>
+      </div>
+
+      {/* Map Viewport & Controls Grid */}
+      <div className="grid grid-cols-1 lg:grid-cols-3 gap-0">
+        {/* Interactive Map Visual Viewport (#3537) */}
+        <div className="lg:col-span-2 relative min-h-[360px] bg-slate-900 flex items-center justify-center p-6 text-white border-b-2 lg:border-b-0 lg:border-r-2 border-black select-none">
+          {/* Map Overlay Canvas Simulation */}
+          <div data-testid="parking-map-canvas" className="w-full h-full border-2 border-dashed border-emerald-500/40 rounded-xl bg-slate-800/80 p-4 relative flex flex-col justify-between">
```

ESLint

[error] Replace `·data-testid="parking-map-canvas"·className="w-full·h-full·border-2·border-dashed·border-emerald-500/40·rounded-xl·bg-slate-800/80·p-4·relative·flex·flex-col·justify-between"` with `⏎············data-testid="parking-map-canvas"⏎············className="w-full·h-full·border-2·border-dashed·border-emerald-500/40·rounded-xl·bg-slate-800/80·p-4·relative·flex·flex-col·justify-between"⏎··········`

(prettier/prettier)